apps/meeting/serverless/src/handlers.js

```diff
@@ -110,21 +124,32 @@ exports.createMeeting = async (event, context, callback) => {
   }
 
   let meetingInfo = await getMeeting(title);
+  let client = getClientForMeeting(meetingInfo);
   if (!meetingInfo) {
-    const request = {
+    let request = {
```

Comment:
I think we don't need to change it to `let`, right?

Reply:
request will be modified from L135 to L143

Reply:
You can still change the property of `request` when using `const`. Because the reference of `request` doesn't change.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const